all-tests/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.jboss.tools</groupId>
 		<artifactId>aerogear</artifactId>
-		<version>1.1.0-SNAPSHOT</version>
+		<version>1.2.0-SNAPSHOT</version>
```

Comment:
This is included in #118. I asked Gorkem to review #118 before we release it to master.